auth/api/http/groups/endpoint.go

```diff
@@ -243,9 +264,10 @@ func buildGroupsResponseTree(page groups.GroupPage) groupPageRes {
 
```

Comment:
I would use `children` here instead of `ch` as you do at the end of this function. `ch` is easily confused with channel and we don't see that it's plural.